src/index.spec.ts

```diff
@@ -0,0 +1,168 @@
+import 'mocha';
+import { assert } from 'chai';
+import { App, ExpressReceiver } from './index';
+import serverless from 'serverless-http';
+import {
+  delay,
+  createFakeMessageEvent,
+  createEventRequest,
+  importAppWithMockSlackClient,
+} from "./test-helpers";
+
+// How I'd like to write apps using koa-style middleware
```

Comment:
this should be a separate conversation/PR, IMO. just make this as simple as possible, like from their docs simple with a bit of async.

Reply:
Possibly, I moved it to the bottom of the file as it made more sense there anyway. Possibly a gist makes more sense.